Packs/Mattermost/Integrations/MattermostV2/README.md

```diff
@@ -651,6 +707,223 @@ No permissions channel.
 
 There is no context output for this command.
 
+### mattermost-list-groups
+
+***
+Lists groups. Groups are available on Enterprise and Professional plans. Minimum server version: 5.11
+
+#### Required Permissions
+
+No permissions required.
+
+#### Base Command
+
+`mattermost-list-groups`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| page | The page number to retrieve. Default value is 0. | Optional | 
+| page_size | The size of the page to retrieve. Default value is 50. | Optional | 
+| limit | How many results to retrieve. Will override the page and page_size arguments if given. | Optional | 
+| group | Search for a specific user by this pattern. | Optional | 
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Mattermost.Groups.id | String | The ID of the group. | 
+| Mattermost.Groups.create_at | Unknown | When was the group created. | 
+| Mattermost.Groups.update_at | Unknown | When was the group updated. | 
+| Mattermost.Groups.delete_at | Unknown | When was the group deleted. | 
```

Comment:
Are these datetimes? if so, the description should be: The datetime the group was.....